app/src/main/res/values/attrs.xml

```diff
@@ -21,6 +21,7 @@
         <attr name="controlIconBackground" format="reference|color" />
         <attr name="controlIconForegroundActive" format="reference|color" />
         <attr name="controlIconForegroundInactive" format="reference|color" />
+        <attr name="button_default_normal_background" format="reference|color" />
```

Comment:
Attributes should use camelCase, also might be better to add this one below "buttonRounding"